include/vrv/doc.h

```diff
@@ -510,6 +510,11 @@ class Doc : public Object {
      */
     pugi::xml_document m_back;
 
+    /**
+     * The music@decls value
+     */
+    std::string m_musicDecls;
```

Comment:
this should be stored as a `pugi::xml_document`

Reply:
@lpugin That's easy to do. Question: it's a bit more than iomei.cpp actually needs; should iomei ignore any sub-elements (since there are already doc->m_back and doc->m_front), and thus only take attributes from doc->m_music?  Or should any sub-elements of doc->m_music be merged with doc->m_back and doc->m_front and place in the output `<music>` element?

Reply:
Alternatively, I could get rid of doc->m_back and doc->m_front, replacing them with doc->m_music (which would then contain any back/front matter).  But that would be a backward-incompatible change, requiring other changes throughout verovio, and might break other folks' code that is based on verovio code.

Reply:
Sorry, ignore my previous comment. I thought you were dealing with a sub-tree, not just an attribute.